packages/lesswrong/server/scripts/importEAGUserInterests.ts

```diff
@@ -3,10 +3,13 @@ import Papa from 'papaparse';
 import Users from '../../lib/collections/users/collection';
 import { updateMutator, Vulcan } from '../vulcan-lib';
 import { wrapVulcanAsyncScript } from './utils';
+import UserEAGDetails from '../../lib/collections/userEAGDetails/collection';
 
 /**
  * Given a CSV with users' Email, Experience, and Interest data from EAG,
- * import it to the users' "experiencedIn" and "interestedIn"
+ * import it to the users' "experiencedIn" and "interestedIn".
+ *
+ * TODO: Prob no longer needed, delete
```

Comment:
I'm still not sure where we'll land with the EAG data so I'll keep this here now and plan to clean up this file later